packages/gatsby-theme-api-docs/src/design-system.js

```diff
@@ -35,5 +41,7 @@ export const typography = {
     th: designSystem.pxToRem('21px'),
     methodType: designSystem.pxToRem('26px'),
     methodTitle: designSystem.pxToRem('32px'),
+    responseCode: designSystem.pxToRem('16px'),
+    responseBodyType: designSystem.pxToRem('19px'),
```

Comment:
Same here, this looks like a magic number. Please change

Reply:
I got it from the design.